src/ray/raylet/agent_manager.cc

```diff
@@ -206,9 +185,7 @@ void AgentManager::CreateRuntimeEnv(
       serialized_allocated_resource_instances);
   runtime_env_agent_client_->CreateRuntimeEnv(
       request,
-      [this,
-       job_id,
-       serialized_runtime_env,
+      [serialized_runtime_env,
```

Comment:
```suggestion
      [serialized_runtime_env=std::move(serialized_runtime_env),
```
Is it possible to do this?

Reply:
We can't do this because `serialized_runtime_env` is a reference from task_spec.